test/fixtures/petStore.no_host.json

```diff
@@ -0,0 +1,1051 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "This is a sample server Petstore server.  You can find out more about Swagger at [http://swagger.io](http://swagger.io) or on [irc.freenode.net, #swagger](http://swagger.io/irc/).  For this sample, you can use the api key `special-key` to test the authorization filters.",
```

Comment:
Agree with @bajtos here. `spec` can be altered for the test before initiating connector.